lib/rsolr/xml.rb

```diff
@@ -16,6 +16,7 @@ def initialize(doc_hash = {})
       doc_hash.each_pair do |field,values|
         # create a new field for each value (multi-valued)
         # put non-array values into an array
+        values = Array(values) if values.respond_to?(:to_a) || values.respond_to?(:to_ary)
```

Comment:
Can we do this unconditionally? I suspect all this logic is a hold-over from Ruby 1.8 days?